Description: fix MIPS-specific source path
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/merge_requests/43
Last-Update: 2024-11-01
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/webrtc/common_audio/meson.build
+++ b/webrtc/common_audio/meson.build
@@ -99,7 +99,7 @@
     'signal_processing/min_max_operations_mips.c',
     'signal_processing/resample_by_2_mips.c',
     'signal_processing/vector_scaling_operations_mips.c',
-    'third_party/ooura/fft_size_128/ooura_fft_mips.c',
+    'third_party/ooura/fft_size_128/ooura_fft_mips.cc',
     'third_party/spl_sqrt_floor/spl_sqrt_floor_mips.c',
   ]
 endif
